### PR TITLE
Create "Isolasi Mandiri" content on homepage

### PR DESCRIPTION
**Pull request by adrianpdm**
### Task Description
[Membuat section Isolasi Mandiri](https://trello.com/c/cKhhgjSA/868-web-s3-membuat-section-isolasi-mandiri)

### Changes
- Replace "Tetap Waspada dengan menerapkan AKB" section with "Isolasi Mandiri"
- Create components for each sub-section of "Isolasi Mandiri"
- Set WA and form backlink as env values

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
                 title="Hotline Pikobar"
                 subtitle="Chat WA seputar COVID-19"
                 number="Klik untuk chat"
-                href="https://s.id/ChatbotPikobar"
+                href="https://wa.me/6285697391854?text=Halo%20Admin!%20Saya%20ingin%20tanya%20seputar%20PIKOBAR"
```
Comment (adzharamrullah) [inline pages/index.vue:29]: **Low priority**
I see at above you define env var the phone number, and why this phone number is not put into and using env var

File: components/_pages/index/IsolasiMandiri/ExpandableContent.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      'exp-content': true,
+      'exp-content--open': mOpen,
+    }"
+  >
+    <h3 class="exp-content__header">
+      <strong>
+        <slot name="title" />
+      </strong>
+      <button class="p-2 ml-8" @click="toggle">
+        <svg
```
Comment (adzharamrullah) [inline components/_pages/index/IsolasiMandiri/ExpandableContent.vue:13]: **Low priority**

You can make the svg script into component for reduce your line in a one page and reuse svg-icon without copying all the script

File: pages/index.vue (hunk 1 above)
Comment (adrianpdm) [reply pages/index.vue:29]: thank you, will fix this asap

File: components/_pages/index/IsolasiMandiri/ExpandableContent.vue (hunk 2 above)
Comment (adrianpdm) [reply components/_pages/index/IsolasiMandiri/ExpandableContent.vue:13]: thank you, will fix this asap

**Review by Ibwedagama — APPROVED** [review]

**Review by yoslie — APPROVED** [review]

**Review by bangunbagustapa — APPROVED** [review]

**Review by Arif9878 — APPROVED** [review]

**Review by naufalihsank — APPROVED** [review]

**Pull request merged**